### ISSUE TITLE
ZGP (Zigbee Green Power) Green Power Sink (GPS) support in zigpy-espzb radio library for ESP32-H2 and ESP32-C6?

### ISSUE THREAD

**Issue by Hedda** [lhespress/zigpy-espzb#9]
Not tested myself but wondering if this [zigpy-espzb](https://github.com/lhespress/zigpy-espzb) radio library supports Green Power Sink (GPS) and/or forwarding to ZGP controller support with the draft ZGP (Zigbee Green Power) patches for zigpy that are still a work in progress?

https://github.com/zigpy/zigpy/pull/1282

https://github.com/konistehrad/zigpy/tree/zgp

https://github.com/zigpy/zigpy/issues/341

FYI, some of the other reference radio libraries for zigpy has specifically been patched by @konistehrad for GPDF (Green Power Data Frames) handling, see example:

https://github.com/zigpy/zigpy-znp/pull/235

https://github.com/zigpy/bellows/pull/594

https://github.com/zigpy/zigpy-deconz/pull/235

PS: At least the upstream [esp-zigbee-sdk](https://github.com/espressif/esp-zigbee-sdk) contains Zigbee GreenPower examples:

https://github.com/espressif/esp-zigbee-sdk/tree/main/examples/esp_zigbee_greenpower/esp_zigbee_gpd

https://github.com/espressif/esp-zigbee-sdk/tree/main/examples/esp_zigbee_greenpower/esp_zigbee_gpc

**Comment by lhespress** [comment]
@Hedda It doesn't support Green Power Now.

**Comment by Hedda** [comment]
> @Hedda It doesn't support Green Power Now.

zigpy as downstream does not yet have initial ZGP device support until this PR is merged -> https://github.com/zigpy/zigpy/pull/1282

But Espressif's esp-zigbee-sdk does support ZGP upstream or? https://github.com/espressif/esp-zigbee-sdk

Again, there are at least examples that seem to indicate that is the case? See:

https://github.com/espressif/esp-zigbee-sdk/tree/main/examples/esp_zigbee_greenpower/esp_zigbee_gpd

and

https://github.com/espressif/esp-zigbee-sdk/tree/main/examples/esp_zigbee_greenpower/esp_zigbee_gpc

**Comment by lhespress** [comment]
@Hedda Yes, the SDK has supported it, i mean the NCP firmware doesn't support.
